internal/pkg/template/templates/environment/partials/environment-manager-role.yml

```diff
@@ -212,7 +212,8 @@ EnvironmentManagerRole:
           Action: [
             "ec2:DescribeSubnets",
             "ec2:DescribeSecurityGroups",
-            "ec2:DescribeNetworkInterfaces"
+            "ec2:DescribeNetworkInterfaces",
+            "ec2:DescribeRouteTables"
```

Comment:
These changes should be fine since it's strictly READ permissions, but we should call it out today while reviewing with AppSec